File: src/main/java/ru/openbank/meetup/demo/java/slide6/ApplicationField.java
package ru.openbank.meetup.demo.java.slide6;

import lombok.Data;

import java.util.List;

@Data
public class ApplicationField {

    private String name;

    private String type;

    private String regEx;

    private boolean required;

    private List<String> postProcessors;

    private String ordersPath;
}

File: src/main/kotlin/ru/openbank/meetup/demo/kotlin/slide6/ApplicationField.kt
package ru.openbank.meetup.demo.kotlin.slide6

data class ApplicationField(
    var name: String? = null,
    var type: String? = null,
    var regEx: String? = null,
    var isRequired: Boolean = false,
    var postProcessors: List<String>? = null,
    var ordersPath: String? = null
)
